### DISCUSSION TITLE
European dimension for nlnet

### DISCUSSION THREAD

**Discussion by damooo2** [manomayam/manas#47] [General]
Hello @hexagon6, @MarcAntoine-Arnaud 

I had applied for nlnet funding to develop Manas. They require that an european collaborator is also required. Would anu of you be willing?

Other than funding for any of your major contributions, you will not be troubled with any hassles regarding reporting, banking, financial hassles.

Would be grateful 🙏🏿

**Comment by damooo** [comment]
@MarcAntoine-Arnaud
Can you help in this regard? Thank you 🙏🏿.

**Comment by MarcAntoine-Arnaud** [comment]
Hi @damooo ,

We need to discuss it, I'm open to help manas project !
Can you contact me at [marc-antoine.arnaud@luminvent.com](mailto:marc-antoine.arnaud@luminvent.com) to plan a visio and discuss more about perspective, collaboration etc.
And we may also discuss if we need to include more people in it ? I may have some first ideas.

Talk to you soon !

**Comment by hexagon6** [comment]
Hi @damooo2 I would be onboard. How can I help? Let me know via <redacted>
